File: canvas/fizz-11/lib/students.js
const axios = require('./axios')

const getStudentsForCourse = async (courseId) => {
  // role_id 3 is the id for student role
  const { data } = await axios.get(
    `/api/v1/courses/${courseId}/users?enrollment_role_id=3`)

  return data
}

const getEnrollmentsForStudent= async (userId) => {
  // role_id 3 is the id for student role
  const { data } = await axios.get(
    `/api/v1/users/${userId}/enrollments`)

  return data
}
//

module.exports = {
  getStudentsForCourse,
  getEnrollmentsForStudent
}

File: src/canvas/data-extractor-py/src/lib/students.py
from lib.canvasRequest import get

def getStudentsForCourse(courseId):
    students = get(f"/api/v1/courses/{courseId}/users?enrollment_role_id=3")
    return students

def getEnrollmentsForStudent(userId):
    enrollments = get(f"/api/v1/users/{userId}/enrollments")
    return enrollments
